### ISSUE TITLE
[Question] xfer OTT VST3 equivalent for linux systems. Would this be feasible?

### ISSUE THREAD

**Issue by ricardofeynman** [jags111/ComfyUI_Jags_Audiotools#2]
There's a project called Linux Studio Plugins that has a linux based Multiband Compressor https://lsp-plug.in/?page=plugins

From the plugin downloads page "LinuxVST 2.4 (full support)" https://lsp-plug.in/?page=download

I still haven't had success with getting Audiotools to install, and am unsure how xfer OTT is being used in the process, but having checked for a cross-platform or linux clone, the LSP Multiband Compressor is the closest I have found which may be of relevance to expand Audiotools support for linux.

Little more info from KVR https://www.kvraudio.com/product/lsp-multiband-compressor-by-linux-studio-plugins-project

**Comment by jags111** [comment]
yes thanks for the wonderful additional updates and will try to add same after exploring if this can be useful unilaterally for all use cases.
I will keep you posted here and if any install issues still exist do post the same.

**Comment by jags111** [comment]
Possibly also still thinking of adding much better UI functionality to the UI so that people will be able to interact much better . This will mean i will have to rewrite all the Java code block to comfy UI so it will  render the graphics much better.. I still have a ton of testing on those.. which I added first and then rebuilding it.

Thanks and do suggest  what will be the impact of same for the windows and mac users and will it give any install or library issues.. as i will have to add a bunch to the requirements.txt.

**Comment by ricardofeynman** [comment]
Thanks I'm looking forward to seeing how it works presently and will provide feedback when I've had a chance to test it. 

The same import failure still persists with the newest commit, but I appreciate you're still building this. I'll check back in the days ahead for new developments.
